### DISCUSSION TITLE
Instalação do Mapos num servidor remoto

### DISCUSSION THREAD

**Discussion by Fernando-Martins-Leal** [RamonSilva20/mapos#2324] [General]
Pessoal, primeiro parabéns pelo time de desenvolvimento pelo projeto tão bacana e ainda gratuito.

Então, a questão agora é enviar a instalação do localhost para um servidor web, que não tenho conseguido nesta versão nova.
Consegui instalar a versão nova no localhost e inclusive consegui fazer alterações incluindo registros e produções de equipamentos, semelhante à função de OS, que posso compartilhar com vcs. É uma necessidade nossa, já que propomos fabricar eletrônicos de IOT a partir de estoque de produtos e serviços de montagens.

No ambiente Localhost rodou suave. Mas quando envio todos os arquivos e crio o banco de dados no servidor de hospedagem, mesmo alterando o route.php e database.php, gera este erro:
"mysqli::real_connect(): (HY000/1045): Access denied for user '1912624_iot'@'185.176.40.75' (using password: YES)"
Vide arquivo anexo.

Já revisei o login do banco de dados, user, nome do banco, senha no servidor, tudo ok.

O que pode ter acontecido? Será que há algum esquema próprio de instalação no servidor, que gere alguma criptografia própria, ou número de api?

Os arquivos que subi e que estão no localhost são os originais do mapos, sem minhas inclusões.

![erro mapos](https://github.com/RamonSilva20/mapos/assets/60037389/d625be59-fbbe-4a8e-a52d-58fcc9c250b5)

**Comment by Fernando-Martins-Leal** [comment]
Opa! Consegui. Tinha um erro de conexão entre o Netbeans e o servidor host. Eu atualizava o login no database, mas ele não subia adequadamente.

Vou agora atualizar as implementações que fiz para incluir produções de equipamentos. Caso alguém se interesse, compartilho aqui depois.

**Reply by patriciolvsilva** [reply]
Eu tenho interesse! Inclusive se tiver tbm a opcao de assinar a OS Digital tbm agradeço! email: lopat@outlook.com

**Reply by hebert-abreu** [reply]
Olá! Tudo bem? Estou com este problema até em localhost, na minha máquina local antes mesmo de subir para alguma hospedagem. Estou utilizando o MySQL 8 com a opção de compatibilidade de senha com MySQL 5 ativada. Por favor nos conte como conseguiu corrigir. me ajude!!!! Obrigado!!!!

**Reply by MateusPFOliveira1996** [reply]
Olá, eu tenho interesse. Se puder enviar agradeço a gentileza mateusscarlett@gmail.com
